### DISCUSSION TITLE
Is IPv6 on the roadmap?

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#1748] [General]
I was looking through the code, and also read somewhere on the site that IPv6 wasn't currently supported.
Seems it could resolve a lot of legacy NAT issues that prevent some from running a server due to ISP limitations.
Is IPv6 support on the roadmap?

**Comment by npostavs** [comment]
There's some work on it: #1017

**Comment by ann0see** [comment]
IPv6 basic support will be in the next version. Please see https://github.com/jamulussoftware/jamulus/discussions/1950

**Comment by JaminShanti** [comment]
I work in the industry, who actually uses IP6?  IP6 is more of a I wish it was used.  I don't think it's usage is going to "sneak up on you" at this rate.  Even for UDP transfer software, like Riverbed, IP4 seems fine.

**Reply by gilgongo** [reply]
We've had a few people say they would use it (on latency/performance grounds I think), but it'll be interesting to see to what extent it would get used, certainly,

**Reply by ann0see** [reply]
Germany uses IPv6 GCN and some people can’t run Jamulus servers because they only have a public IPv6  address

**Reply by JaminShanti** [reply]
I'll take your word on it.  I am not currently on a public site gig.  The last time I worked on a public site, saw very little dns  lookup traffic on IP6.
